CHANGES.md

```diff
@@ -3,6 +3,8 @@ Change Log
 
 ### 1.23 - 2016-07-01
 
+* Added `heightReference` property to point, billboard and model entities that would allow them to be rendered relative to terrain.
+* Changed corridor, polygon and rectangle entities to conform to terrain by using a `GroundPrimitive` if they are filled and don't have `height` or `extrudedHeight`.
```

Comment:
You also need to mention KML now clamps to terrain by default and GeoJSON was also udpated but needs to be enabled with `clampToGround`.  Also, specifically mention the new CZML properties and write up a czml-writer issue (or open PR) so they get added to the writer and CZML doc as well.